vesc_ackermann/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ elseif("${CMAKE_CXX_STANDARD_COMPUTED_DEFAULT}" STREQUAL "98")
 endif()
 
 find_package(ament_cmake_auto REQUIRED)
+find_package(ackermann_msgs REQUIRED)
```

Comment:
Since this uses `ament_cmake_auto`, there is no need to add this to `CMakeLists.txt`. If you add it to `package.xml` as a `<depend>`, then it will automatically link against it.